tests/TestCase.php

```diff
@@ -12,6 +13,20 @@ abstract class TestCase extends BaseTestCase
     protected function setUp(): void
     {
         parent::setUp();
+
+        $this->withoutVite();
+    }
+
+    protected function getPackageProviders($app): array
+    {
+        return [
+            ActivityLogServiceProvider::class,
+        ];
     }
 
+    protected function useDatabase(): void
+    {
+        $this->app->useDatabasePath(__DIR__ . '/./Database');
```

Comment:
i think the path should be __DIR__ . '/../tests/Database'

Reply:
Okay. Thank you for correcting my work, good sir.

Reply:
Do you both know what __DIR__ does?